Contributing.md

```diff
@@ -8,7 +8,7 @@ We warmly welcome contributions to the Akeyless GitHub Actions Plugin! Whether i
 Fork this repository on GitHub and clone your fork locally.
 
 ```
-git clone [your-fork-url]
+git clone https://github.com/akeyless-community/akeyless-github-action.git
 cd [repository-name]
```

Comment:
WDYT changing the repository-name to `akeyless-github-action` (or the default one that is been created when doing the clone like you wrote)

Reply:
@amirmao changing to akeyless-github-action like ex: `git clone akeyless-github-action ? `

Reply:
`cd akeyless-github-action`